Backpacks.cs

```diff
@@ -526,6 +527,8 @@ private class Backpack
             private ItemContainer _itemContainer = new ItemContainer();
             private List<BasePlayer> _looters = new List<BasePlayer>();
 
+            public bool hasOverflowingItems = false;
```

Comment:
This should likely be private and respect the private field naming convention.

Reply:
Will update.